steward/src/cellars.rs

```diff
@@ -228,14 +236,15 @@ pub fn validate_cache_price_router(
     cellar_id: &str,
     check_total_assets_value: bool,
     allowable_range_value: u32,
+    expected_price_router: Option<&str>,
 ) -> Result<(), Error> {
-    if !check_total_assets_value || allowable_range_value != 500 {
-        return Err(sp_call_error(
-            "unauthorized arguments for cachePriceRouter call".to_string(),
-        ));
-    }
     let cellar_id_normalized = normalize_address(cellar_id.to_string());
-    if !ALLOWED_CACHE_PRICE_ROUTER.contains(&cellar_id_normalized.as_str()) {
+    let args = (
+        check_total_assets_value,
+        allowable_range_value,
```

Comment:
The requirements description had these new cache price router calls as accepting `<=500` rather than the previous expectation of exactly 500. This means that the method of specifying exact lists of static values won't map to the acceptable input values.